websites/O/Overleaf/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+        "name": "Enevevet",
+        "id": "329669021043523594"
+    },
+    "url": "www.overleaf.com",
+    "description": {
+        "en": "Overleaf is a collaborative cloud-based LaTeX editor used for writing, editing and publishing scientific documents. It partners with a wide range of scientific publishers to provide official journal LaTeX templates, and direct submission links.",
+        "fr": "Overleaf est un éditeur LaTeX en ligne, collaboratif en temps réel."
```

Comment:
how can the length differ so much xD

Reply:
Right, I forgot to translate the rest of the string :/
I updated the file and commited!